ts/src/connections/retryingTunnelConnectionEventArgs.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Event args for tunnel connection retry event.
+ */
+export class RetryingTunnelConnectionEventArgs {
+    constructor(
+        /**
+         * Gets the error that caused the retry.
+         */
+        public readonly error: any,
```

Comment:
Can it be `Error` type? If somehow we catch a non-Error error, we should probably wrap it in an `Error`.